docs/README.md

```diff
@@ -89,7 +89,7 @@ spec:
 
 Istio uses a ConfigMap for its global configuration, called the MeshConfig. All of its settings are available through `spec.meshConfig`.
 
-To support canary updates of the control plane, Sail Operator includes support for multiple Istio versions. You can select a version by setting the `version` field in the `spec` to the version you would like to install, prefixed with a `v`. You can then update to a new version just by changing this field.
+To support canary updates of the control plane, Sail Operator includes support for multiple Istio versions. You can select a version by setting the `version` field in the `spec` to the version you would like to install, prefixed with a `v`. You can then update to a new version just by changing this field. An `vX.Y-latest` alias can be used for the latest z/patch versions of each supported y/minor versions. As per the example above, `v1.23-latest` can be specified in the `version` field. By doing so, the operator will keep the istio version with the latest `z` version of the same `y` version. 
```

Comment:
Is this possible only for minor versions? So there is no `latest` alias? How will this affect istio revision name?

Reply:
Do you mean `v1.23.0`? The revision name is not affected. The `version` value in the revision should get processed the same way.

Reply:
> Is this possible only for minor versions? So there is no latest alias?

If i understand correctly, we will no longer support `latest` but will be supporting the following.
`master`
`v1.24-latest`
`v1.23-latest` 
...


And IstioRevision would point to the actual version.

Reply:
This should be also documented here. Users will have the same questions.

Reply:
Given that this is pretty important, I actually wonder if there should be a "Version" section in the document (perhaps between Gateways and Update strategy) that describes how this field works in a bit more detail, with more concrete examples spelled out. For example, if I set the version field to "v1.23-latest", what exactly happens if I am on Istio 1.23.2 and 1.23.3 becomes available? I think